integration/experiment/power_sweep/gen_policy_recommendation.py

```
@@ -214,7 +257,75 @@ def policy_min_energy(plrange, enmodel, rtmodel = None, pltdp = None,
                 'energy': best_energy}
 
 
-def main(full_df, region_filter, dump_prefix, min_pl, max_pl, tdp, max_degradation):
+def rootssq(ll):
+    "Return the root of the sum of the squares of the list of values."
+    try:
+        return sum([ss**2  for ss in ll])**0.5
+    except TypeError:
```

why is this exception being ignored, can you add a comment please?